### ISSUE TITLE
Not playing back or resuming when choosing "resume"

### ISSUE THREAD

**Issue by GodsLikelyMe** [123Venom/plugin.video.venom#100]
I set Settings - Player - Play next video automatically and just set Episodes on, so one clicking play would set up "PlayNext", but if I choose "resume" from something I was watching, it wont play the video. It wont do anything. I have to choose "play from beginning" for the video to play. 

Venom log
https://paste.ubuntu.com/p/w8MWgPJ4B2/

Kodi log
https://paste.ubuntu.com/p/qh8Vt4FhXD/

**Comment by ShaVerZ** [comment]
I had this issue a few days ago, off the top of my head, I turned off Sync playback progress to Kodi (or similar) within the Trakt addon - this fixed my issue.On 6 Jul 2021 16:30, GodsLikelyMe ***@***.***> wrote:
I set Settings - Player - Play next video automatically and just set Episodes on, so one clicking play would set up "PlayNext", but if I choose "resume" from something I was watching, it wont play the video. It wont do anything. I have to choose "play from beginning" for the video to play.
Venom log
https://paste.ubuntu.com/p/w8MWgPJ4B2/
Kodi log
https://paste.ubuntu.com/p/qh8Vt4FhXD/

**Comment by 123Venom** [comment]
@GodsLikelyMe just looking at your log and I'm curious as to why you are still on the test version and not official release that is more current?  However that will more than likely not fix this issue as the resume point method is untouched code in latest release.
The error from your log implies to me you are not using Trakt scrobbles?  Trakt stores it's scrobble pause as a watched percentage, vs. local resume that stores as a time in seconds.  I have to calc that percentage back to a time in seconds and it's returned as a float().  Your log is showing a string in seconds and that occurs when pulled from the local database when not using Trakt Scrobbles.  The error is cause from the new **yesnocustom** dialog I added to allow you to cancel it at that point without doing anything.  That returns a '-1'.  If the returned string, in your case is '1193.8570556640625' int() on that will throw that error.  I need to switch that line 81 to **int(float(self.offset)** of simply just make my cancel check **if self.offset = '-1'** lol. Simple fix.  Problem is when it encounters the error it throws the Exception and does not play. Trakt scrobble users don't see the issue because it's already returned as a float() due to the conversion from percentage to time in seconds.  Sorry I babble I know.

I'll fix that in next update.  btw...if you only have one device I do suggest to people to stay on local resume.  If you have multiple devices you want those resume points to be in sync then I'd switch to Trakt scrobble within Venom.

**Comment by GodsLikelyMe** [comment]
@123Venom not sure why it would say I'm the test version. I have been officially updating since 6.0. As for the other stuff, I'm lost but know you know so I'll just patiently await any kind of update. 

I'll try a uninstall/install to get a fresh version installed.

**Comment by 123Venom** [comment]
@GodsLikelyMe for a quick test I switched my own setup from Trakt scrobble to local bookmarks.  Upon trying to resume..nada

msg : ValueError -> invalid literal for int() with base 10: '446.9250183105469'

I just changed the cancel check and problem gone.  I just forgot all about local bookmarks return that as a string while Trakt scrobble returns a float.

**Comment by 123Venom** [comment]
> 
> 
> @123Venom not sure why it would say I'm the test version. I have been officially updating since 6.0. As for the other stuff, I'm lost but know you know so I'll just patiently await any kind of update.
> 
> I'll try a uninstall/install to get a fresh version installed.

The issue might be since that test version was installed from a zip Kodi Matrix no longer handles auto updates.  You then have to install from the repo.  A change they made to Matrix but if you use the Venom setting to inform you of updates I flash a notification of this update upon startup so you know about it.

**Comment by GodsLikelyMe** [comment]
@123Venom  Thank you for getting back and working out issues. You are a beast! 
I updated to 6.0 by the repo in kodi. At first the repo didnt list the new version, so I made it check for updates and thats when it showed the new version and then I installed it. Did the same thing up to 6.0.2. 

Either way, I've already uninstalled  it and reinstalled for a fresh copy. 
Everything else has been working great. Thank you for the work!
